src/EnergyPlus/DataTimings.cc

```diff
@@ -271,6 +271,8 @@ namespace DataTimings {
 #ifdef EP_Timings
         int loop;
 
+        char const tabchar = '\t';
```

Comment:
Not constexpr?

Reply:
I didn't change it. The `EP_Timings` preprocessor directive is totally broken at this point, so someone should fix that up separately. Maybe we need to add that to our alternate build config, or maybe it all just needs to go away since we're obviously not using it